### PR TITLE
Add ta view

### PR DESCRIPTION
**Pull request by RayOconnor**
I made the base for the Ta View.  It displays relevant pending courses for the TA that is signed in

### CODE REVIEW

File: submit_reports/models.py [hunk 1]
```diff
@@ -80,6 +81,12 @@ def __unicode__(self):
 
 ###############################################################
 
+class StaffManager(models.Manager):
+	def create_student(self, user, nuid, grad_year):
+		staff = self.create(user=user)
+		staff.user = user
+		return student
```
Comment (RayOconnor) [inline submit_reports/models.py:96]: @vogel-sa what is the purpose of this code?

File: submit_reports/models.py [hunk 2]
```diff
@@ -106,7 +116,12 @@ def __unicode__(self):
 # Data Classes
 ######################################################
 
+class SubmitReportManager(models.Manager):
+	def query_pending_reports(request):
+		return SubmitReport.objects.all().filter(status="PENDING")
```
Comment (RayOconnor) [inline submit_reports/models.py:119]: according to this link we should be adding queries to managers. @msuk how do you feel about this?

File: submit_reports/views.py [hunk 3]
```diff
@@ -112,6 +111,47 @@ def faculty_view(request):
...
+
+	temp = Template("""<form method='POST' action=''>
+		{% csrf_token %}
+		{{form.as_p}}
+		<input type='submit' value="Search", action="">
+		</form>"""
+		+ '\n' + table)
+	template = Tplate(temp)
+	context = Context({'form': form,})
+	return HttpResponse(template.render(context=context, request=request))
```
Comment (RayOconnor) [inline submit_reports/views.py]: We should probably look into abstracting this code away. @msuk @vogel-sa do you guys have any ideas?

File: submit_reports/models.py (hunk 1 above)
Comment (vogel) [inline submit_reports/models.py]: @RayOconnor I have no idea

**Pull request merged**